crates/biome_js_analyze/src/lint/correctness/use_jsx_key_in_iterable.rs

```diff
@@ -56,19 +68,29 @@ declare_node_union! {
     pub ReactComponentExpression = JsxTagExpression | JsCallExpression
 }
 
+#[derive(Debug, Default, Clone, Serialize, Deserialize, Deserializable, Eq, PartialEq)]
+#[cfg_attr(feature = "schemars", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase", deny_unknown_fields, default)]
+pub struct UseJsxKeyInIterableOptions {
+    check_shorthand_fragments: bool,
```

Comment:
These options are exposed to our users via the JSON schema, so we need to provide documentation via doc comments (`/// It checks...`)